package.json

```diff
@@ -26,10 +26,13 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "eslint": "^5.15.3",
-    "lerna": "^3.16.4",
     "jsdom": "^14.0.0",
+    "lerna": "^3.16.4",
     "mocha": "^6.1.4",
     "nyc": "^14.0.0",
     "rimraf": "^2.6.3"
+  },
+  "dependencies": {
```

Comment:
I can fix this up, but this will need to go into the CLI's _package.json_.  For Lerna repos, to install dependencies per package, you just go to into the root folder of that package where _package.json_ is and just run the command, e.g
```shell
$ yarn add fs-extra
```